BankSDK/GiniBankSDK/Pod/README.md

```diff
@@ -4,12 +4,28 @@ Distributing Gini Bank SDK XCFrameworks using CocoaPods
 How to release new versions
 ---------------------------
 
-1. After the `.xcframeworks` have been created compress them into a zip named `GiniBankSDK-XCFrameworks.zip`.
-2. Update the version in the `GiniBankSDK.podspec`, for example `spec.version = 2.2.1`.
-3. Clone the `https://github.com/gini/gini-podspecs` repository.
-4. Go into the `gini-podspecs` repository folder and create a subfolder in `GiniBankSDK` which is named after the version, for example `GiniBankSDK/2.2.1`.
-5. Copy the `GiniBankSDK-XCFrameworks.zip` and the `GiniBankSDK.podspec` into the new folder.
-6. Commit and push the new folder and its contents.
+1. After the `.xcframeworks` have been created compress them into a zip named `GiniBankSDK-XCFrameworks.zip`.  
```

Comment:
[suggestion] We can change the steps starting with step 2 with:
2. Clone the `https://github.com/gini/gini-podspecs` repository.
3. Go into the `gini-podspecs` repository folder and create a subfolder in `GiniBankSDK` which is named after the version, for example `2.2.1`.
4. Copy the `GiniBankSDK-XCFrameworks.zip` into the new folder.
5. Copy the `GiniBankSDK.podspec` file from a previous version and update the version with the corresponding one, for example `spec.version = 2.2.1`.
6. Commit and push the new folder and its contents to `gini-podspecs` repo.

Reply:
Awesome! I only have one comment regarding point 5: The reason to copy the `GiniBankSDK.podspec` file from this folder (where the README.md is located) was to include any changes we made to the `.podspec` file since the previous release. For example we could have updated the license or the year or the ios deployment target in the `.podspec` file since the previous release. 

Are you ok with changing point 5 from your suggestion to this:
```
5. Copy the `GiniBankSDK.podspec` file from the folder where this readme is located and update the version with the corresponding one, for example `spec.version = 2.2.1`.
```

Reply:
Yes, sounds good. Thanks!